src/plugins/home/public/application/components/_home.scss

```diff
@@ -1,3 +1,28 @@
+// Local page vars
+$homePageHeaderHeight: $euiSize * 8;
+$homePageWidth: 1200px;
+
+.homPageHeader {
+  height: $homePageHeaderHeight;
```

Comment:
Is it necessary to define a fixed height for the home page header here? Ideally, the height of the header would scale to its contents. If only being used to account for the `min-height` in `.homPageContainer`, perhaps we can use a flexbox solution instead?

Reply:
This is also worth tinkering with. This fixed height made it simpler, in the end, to account for some of the shaded background color that was showing at the bottom of the screen. In other words, I was able to also use this value to resolve that issue. That said, it might be the culprit that generated the bottom 'band' 🤔